core/src/test/java/nl/tudelft/contextproject/core/rendering/DrawablePixmapTest.java

```diff
@@ -0,0 +1,130 @@
+package nl.tudelft.contextproject.core.rendering;
+
+import com.badlogic.gdx.graphics.Pixmap;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.Vector2;
+import nl.tudelft.contextproject.core.config.Constants;
+import nl.tudelft.contextproject.core.entities.Colour;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.*;
+
+public class DrawablePixmapTest {
+
+    protected Pixmap painting;
+    protected int pixel;
+
+    protected Pixmap newPainting;
+    protected int newPixel;
+
+    protected Texture texture;
+    protected DrawablePixmap drawablePixmap;
+
+    @Before
+    public void setUp() {
+        pixel = Colour.RED.getPixelValue();
+        newPixel = Colour.YELLOW.getPixelValue();
+
+        painting = mock(Pixmap.class);
+        newPainting = mock(Pixmap.class);
+        texture = mock(Texture.class);
+
+        drawablePixmap = new DrawablePixmap(painting, newPainting, texture);
+        when(painting.getPixel(0, 0)).thenReturn(pixel);
+        when(newPainting.getPixel(0, 0)).thenReturn(newPixel);
+    }
+
+    @Test
+    public void constructorTest() {
+        assertTrue(drawablePixmap != null);
```

Comment:
this doesn't test anything, even if the constructor misbehaves this will always be true.